roles/openshift_health_checker/openshift_checks/sdn.py

```diff
@@ -240,17 +251,21 @@ def check_node(self):
         if self.want_full_results:
             try:
                 service_prefix = self.get_var('openshift_service_type')
-                if self._templar is not None:
```

Comment:
I remember when this went in, we added this line to unbreak the unit tests.  Probably need to mock something up in the tests if that is failing (or if they are even still run for this file?)

Reply:
I copied the pattern from other checks, but I don't remember whether I ever saw a case where `self._template` was `None`.  https://github.com/openshift/openshift-ansible/commit/edde1f2bf0fa180fc69d905ce2aa27e68f7295dd introduced the `template_var` method and says it is "mostly for unit tests", but is there a reason not to use it generally?

(My preference would be for `get_var` to handle it, possibly optionally, but that would be an invasive change.)

Reply:
I think you have inadvertently stumbled upon a sea of technical debt in this file.

In any case, your patch is not failing current unit tests, I think this used to be run via the 'tox' CI job we have, which seems to be currently disabled.

If it is passing current CI tests, no need to re-add it.